src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ApproverCaseloadServiceTest.kt

```diff
@@ -0,0 +1,96 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.caseload
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.reset
+import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.LicenceSummaryApproverView
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.PrisonApproverService
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceKind
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class ApproverCaseloadServiceTest {
+  private val prisonApproverService = mock<PrisonApproverService>()
+
+  private val service = ApproverCaseloadService(prisonApproverService)
+
+  @BeforeEach
+  fun reset() {
+    reset(prisonApproverService)
+  }
+
+  @Test
+  fun `CADM prison code is filtered out`() {
```

Comment:
to test this you'd need to verify the call to prisonApprover caseload?  and check the CADM caseload isn't passed through